search-app/src/cmr/search/services/humanizers/humanizer_report_service.clj

```diff
@@ -86,26 +86,27 @@
 (defn generate-humanizers-report-csv
   "Returns a report on humanizers in use in collections as a CSV."
   [context]
-  (let [[t1 collection-batches] (u/time-execution
+  (let [[t1 collection-batches] (util/time-execution
                                   (get-all-collections context))
          string-writer (StringWriter.)
          idx-atom (atom 0)]
-    (info "get-all-collections:" t1
-           "processing " (count collection-batches)
-           " batches of size" (humanizer-report-collection-batch-size))
+    (info (format "get-all-collections: %d ms. Processing %d batches of size %d"
```

Comment:
I haven't looked, but does CMR support the standard logging `infof`, `debugf`, etc., functions? If so, we can use those instead of wrapping a format call (... I've been meaning to look into that).